app/service/file_svc.py

```diff
@@ -96,6 +124,22 @@ def __init__(self, plugins, exfil_dir):
         """
         self.special_payloads[name] = func
 
+    async def find_payloads(self):
+        """
+        Identify the full gamut of available payloads (filtering out adversary mode)
```

Comment:
why filter out adversary mode plugin? we should look in any plugin directory.

Reply:
Any payload stored in Adversary plugin is not usable by Chain (custom XORing of the files that is only undone when the adversary mode plugin is running an operation).